### PR TITLE
#add handlers for fly update layers

### PR DESCRIPTION
**Pull request by ipostol**
Hello, this is update for resolve fly updates layerTypes by new props (now they can inititialize with map 1 time)

### CODE REVIEW

File: src/google_map.js [hunk 1]
```diff
@@ -291,6 +294,14 @@ export default class GoogleMap extends Component {
 
         this.map_.setOptions(options);
       }
+
+      if (!isEqual(nextProps.layerTypes.sort(), this.props.layerTypes.sort())) {
```
Comment (istarkov) [inline src/google_map.js]: imo it's on user to sort layers, and to support referential equality. I don't like any kind of isEqual + sort checks here.

**Comment by ipostol** [comment]
@istarkov, remove sort/isEqual logic, left only ref equal

**Comment by istarkov** [comment]
Thank you!

**Comment by istarkov** [comment]
0.21.3

**Pull request merged**